simulation_parameters/microbe_stage/organelles.json

```diff
@@ -208,7 +208,7 @@
     "ConsumptionColour": "#ff7248",
     "IconPath": "res://assets/textures/gui/bevel/parts/ChemoproteinsIcon.png",
     "EditorButtonGroup": "Protein",
-    "EditorButtonOrder": 3,
+    "EditorButtonOrder": 5,
```

Comment:
Why did you reorder buttons in the protein group? This PR adds just one organelle so I would have only expected that category to need space made for inserting that organelle.